src/components/game/components/board/Board.js

```diff
@@ -20,11 +20,16 @@ const Board = ({
   showNextPhaseButton,
   onClickNextPhaseHandler,
   showAiCards,
+  logStats,
 }) => (
   <div className="container">
     <div className="top-player">
       <div className="board-row">
-        <Hand hand={aiHand} visible={showAiCards ? true : false} />
+        <Hand
+          hand={aiHand}
+          visible={showAiCards ? true : false}
```

Comment:
```suggestion
          visible={showAiCards}
```
booleans and predicates return the logical boolean value of true or false so it can be omitted

Reply:
Ok.